airsonic-main/src/main/java/org/airsonic/player/Application.java

```diff
@@ -36,10 +35,17 @@
         MultipartAutoConfiguration.class, // TODO: update to use spring boot builtin multipart support
         LiquibaseAutoConfiguration.class})
 @Configuration
-@ImportResource({"classpath:/applicationContext-service.xml",
-        "classpath:/applicationContext-cache.xml",
-        "classpath:/applicationContext-sonos.xml",
-        "classpath:/servlet.xml"})
+@EnableWebMvc
+@ComponentScan(basePackages = {
+        "org.airsonic.player.ajax",
+        "org.airsonic.player.controller",
+        "org.airsonic.player.dao",
+        "org.airsonic.player.i18n",
+        "org.airsonic.player.monitor",
+        "org.airsonic.player.security",
+        "org.airsonic.player.service",
+        "org.airsonic.player.spring",
+        "org.airsonic.player.validator"})
```

Comment:
Why not replace this with @SpringBootApplication?

It will autoconfigure the ComponentScans. Are we trying to protect some packages from scanning?

Reply:
I'm not necessarily opposed to `@SpringBootApplication` but for now this keeps the functionality as close as to what it was before. Changing to `@SpringBootApplication` is out of scope for this PR.

Reply:
SpringBootApplication is already annotated, like a few lines above this comment, I'm not seeing how it is out-of-scope? It's part of the class declaration.

Removing this will also reduce your PR LoC footprint. The only reason you'd have this here would be if you were trying to protect some packages from scanning, which I don't think we have any of.

Reply:
A separate PR should be considered for component scanning the whole package tree.